triage/labels.rst

```diff
@@ -54,9 +54,19 @@ These labels are mostly used to specify which :ref:`part of the codebase
   (written in Python) and other changes related to tests, :mod:`unittest`,
   or :mod:`doctest`.
 
+OS labels
+=========
 
-Expert labels
-=============
+These labels are used to specify which operating systems are affected.
+Since most issues either affect all systems or are specific to Unix,
+the only available labels are :gh-label:`OS-windows`, :gh-label:`OS-mac`,
+and :gh-label:`OS-freebsd`.
+
+
+.. _Expert labels:
+
+Topic labels
+============
 
 These labels are used to specify the area of expertise required to address
```

Comment:
Maybe worth updating this a bit too, to reflect the title change and avoid implying that being an expert is required to interact with the PR?

```suggestion
These labels are used to denote the specific topic area, if any, of
```